### PR TITLE
Added a test checking the case of filtered away tectonic region types

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1428086, companion of https://github.com/gem/oq-risklib/pull/169. The tests are running here: https://ci.openquake.org/job/zdevel_oq-engine/1078

### CODE REVIEW

File: qa_tests/hazard/classical/test.py [hunk 1]
```diff
@@ -693,3 +694,30 @@ def test(self):
...
+
+        curves = models.HazardCurveData.objects.filter(
+            hazard_curve__output__oq_job=job.id,
+            hazard_curve__statistics='mean'
+        )
+        [pga_curve] = curves.filter(hazard_curve__imt='PGA')
+
+        exported_file = hazard_export.export(
+            pga_curve.hazard_curve.output.id, result_dir, 'csv')
+        actual = [' '.join(line.split(' ')[2:]).strip()
```
Comment (ptormene) [inline qa_tests/hazard/classical/test.py]: Please add a comment

File: qa_tests/hazard/classical/test.py [hunk 2]
```diff
@@ -693,3 +694,30 @@ def test(self):
...
+                  for line in open(exported_file)]
+        fname = os.path.join(os.path.dirname(case_19.__file__), 'expected',
+                             'hazard_curve-mean.csv')
+        expected = [line.split(',')[1] for line in open(fname)]
```
Comment (ptormene) [inline qa_tests/hazard/classical/test.py]: Please add another comment

**Comment by ptormene** [comment]
Apart from the comments, LGTM

**Comment by micheles** [comment]
I have added comments on the export formats.

**Pull request merged**